### PR TITLE
Sendtoaddr confirm

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/sync/update_tx.js [hunk 1]
```diff
@@ -37,6 +38,22 @@ async function updatePendingTxs(db) {
 
 // Update the Transaction info
 async function updateTx(tx) {
+  // sendtoaddress does not use the same confirmation method as EVM txs
+  if (tx.type === 'TRANSFER' && tx.token === 'QTUM') {
+    const txInfo = await wallet.getTransaction({ txid: tx.txid });
+    const blockInfo = await blockchain.getBlock({ blockHash: txInfo.blockhash });
```
Comment (frankobe) [inline src/sync/update_tx.js]: The request to get current block should be called only once per sync. If needed, we can pass it as a parameter in `updateTx(tx, currenBlock, currentBlockTime)`

File: src/qclient.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const { Qweb3 } = require('qweb3');
```
Comment (frankobe) [inline src/qclient.js]: Nice singleton

File: src/config/config.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ const Config = {
   DEFAULT_LOGLVL: 'info',
   CONTRACT_VERSION_NUM: 0,
   TESTNET: true,
+  TRANSFER_MIN_CONFIRMATIONS: 3,
```
[the remark this answers is not in the document]
Comment (dwalintukan) [reply src/config/config.js]: Neil suggested to have 3-6 confirmations for wallet to wallet sends. We have to treat this differently than EVM execution. Contract txs are confirmed in 1 block. Possible that wallet txs might be reversed.

File: src/qclient.js (hunk 2 above)
Comment (dwalintukan) [reply src/qclient.js]: 👍

File: src/sync/update_tx.js (hunk 1 above)
Comment (dwalintukan) [reply src/sync/update_tx.js]: fixed

File: src/config/config.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (frankobe) [reply src/config/config.js]: @dwalintukan it is true that wallet tx can be revered but before reverse, the balance is already updated. This should be in sync with the balance we display so we can only wait for 1 confirmation here

[the remark this answers is not in the document]
Comment (dwalintukan) [reply src/config/config.js]: Actually believe it won't show up in listunspent. Or at least there is a way to specify the min num of confirmations. I think default is 6

[the remark this answers is not in the document]
Comment (dwalintukan) [reply src/config/config.js]: Can check the rpc help for listunspent. If I am correct with default as 6, then we should actually change ours to 6 too.

**Pull request merged**